lib/semanticchess.test.js

```diff
@@ -172,7 +173,68 @@ describe('Semantic Chess', function() {
       });
 
     });
+	
+  });
+  
+  describe('promotion', function() {
+	  
+    it('valid moves', async function() {
+		var semanticChess = new SemanticChess({
+		  url: 'http://example.org/myGame',
+		  moveBaseUrl: 'http://example.org/myfile',
+		  userWebId: 'http://example.org/#me',
+		  opponentWebId: 'http://example.org/#other',
+		  name: 'My game',
+		  uniqid: Utils.customUniqid()
+		});
+		
+		//POSITIVE TESTS
+		//Test in the middle of the board (white pawn)
+		semanticChess.chess = new chess.Chess('8/4P3/8/8/8/8/K6k/8 w - - 0 1');
+		assert.ok(semanticChess.isPromotion('e7','e8'));
+		//Test in one side of the board (white pawn)
+		semanticChess.chess = new chess.Chess('8/P7/8/8/8/8/K6k/8 w - - 0 1');
+		assert.ok(semanticChess.isPromotion('a7','a8'));
+		//Test in one side of the board (black pawn)
+		semanticChess.chess = new chess.Chess('8/7k/8/8/8/8/K6p/8 b - - 0 1');
+		assert.ok(semanticChess.isPromotion('h2','h1'));
+		//If the pawn is capturing another piece
+		semanticChess.chess = new chess.Chess('3n4/4PP1k/7r/8/8/8/K3p3/8 w - - 0 1');
+		assert.ok(semanticChess.isPromotion('e7','d8'));
+  });
+  
+  it('invalid moves', async function() {
+		var semanticChess = new SemanticChess({
+		  url: 'http://example.org/myGame',
+		  moveBaseUrl: 'http://example.org/myfile',
+		  userWebId: 'http://example.org/#me',
+		  opponentWebId: 'http://example.org/#other',
+		  name: 'My game',
+		  uniqid: Utils.customUniqid()
+		});
+		
+		//NEGATIVE TESTS
+		//If the king is in check
+		semanticChess.chess = new chess.Chess('8/4PP1k/8/7r/q7/8/K7/2n4Q w - - 0 1');
```

Comment:
Can you change this to use the Semantic Chess constructor? I know you can directly access the `chess` variable of an instance, but it is cleaner to use the constructor. Also if users check the test code for usage example they might get the wrong idea about how to use the library.